src/server-dev.ts

```diff
@@ -53,6 +53,7 @@ process.nextTick(async () => {
                     },
                 },
                 authentication: {
+                    // type: IAuthType.NONE,
```

Comment:
deliberate?

Reply:
we should not use none developing locally.